File: UI/src/app/pages/booking/components/booking-preparation/components/screening-room/api-models/screening-room-plan-api.model.ts
import {ScreeningRoomPlanRowApiModel} from './screening-room-plan-row-api.model';

export class ScreeningRoomPlanApiModel {
  public id: string;
  public screeningRoomName: string;
  public rows: ScreeningRoomPlanRowApiModel[];

  constructor() {
    this.rows = [];
  }
}

File: API/src/main/java/com/MateuszLebioda/OneCinema/Model/roomPlan/ScreeningRoomPlanApiModel.java
package com.MateuszLebioda.OneCinema.Model.roomPlan;

import java.util.List;

public class ScreeningRoomPlanApiModel {
    private String id;
    private String screeningRoomName;
    private List<ScreeningRoomPlanRowApiModel> rows;


    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getScreeningRoomName() {
        return screeningRoomName;
    }

    public void setScreeningRoomName(String screeningRoomName) {
        this.screeningRoomName = screeningRoomName;
    }

    public List<ScreeningRoomPlanRowApiModel> getRows() {
        return rows;
    }

    public void setRows(List<ScreeningRoomPlanRowApiModel> rows) {
        this.rows = rows;
    }


}
